### ISSUE TITLE
TorchScript/ONNX unsupported constant kind s

### ISSUE THREAD

**Issue by TMVector** [pytorch/pytorch#22969]
## 🐛 Bug

When I try to run this MRE I get the following error:
```
..
~/anaconda3/lib/python3.7/site-packages/torch/onnx/utils.py in _run_symbolic_function(g, n, inputs, env, operator_export_type)
    600                 else:
    601                     raise RuntimeError("Unsupported prim::Constant kind: `{}`. Send a bug report.".format(
--> 602                         n.kindOf("value")))
    603             elif n.mustBeNone() or op_name == "ListConstruct" or op_name == "ListUnpack":
    604                 # None is not an ONNX operator; keep it as None

RuntimeError: Unsupported prim::Constant kind: `s`. Send a bug report.
```

## To Reproduce

1. Run the code snippet
```python
import torch
class Model1(torch.jit.ScriptModule):
    def __init__(self):
        super().__init__()

    @torch.jit.script_method
    def forward(self, x):
        x = torch.nn.functional.pad(x, (0, 99))
        return x

x = torch.randn(32, 1, 500)
model_onnx = torch.onnx._export(Model1(), x, "tuple_value.onnx",
                                verbose=True,
                                operator_export_type=torch.onnx.OperatorExportTypes.ONNX,
                                example_outputs=Model1()(x))
```
## Expected behavior

I would expect no error.

## Environment

PyTorch version: 1.1.0
Is debug build: No
CUDA used to build PyTorch: Could not collect

OS: Ubuntu 18.04.1 LTS
GCC version: (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
CMake version: Could not collect

Python version: 3.7
Is CUDA available: No
CUDA runtime version: Could not collect
GPU models and configuration: GPU 0: GeForce GTX 1060 6GB
Nvidia driver version: 410.66
cuDNN version: Could not collect

Versions of relevant libraries:
[pip] numpy==1.16.2
[pip] numpydoc==0.8.0
[pip] torch==1.1.0
[pip] torchvision==0.3.0
[conda] blas                      1.0                         mkl  
[conda] mkl                       2019.3                      199  
[conda] mkl-service               1.1.2            py37he904b0f_5  
[conda] mkl_fft                   1.0.10           py37ha843d7b_0  
[conda] mkl_random                1.0.2            py37hd81dba3_0  
[conda] pytorch-cpu               1.1.0               py3.7_cpu_0    pytorch
[conda] torchvision-cpu           0.3.0             py37_cuNone_1    pytorch

**Comment by TMVector** [comment]
Hi @houseroad, just wondered if you'd had a chance to look at this? :slightly_smiling_face:

**Comment by kwanUm** [comment]
+1, happened to me as well.

**Comment by v1093** [comment]
+1, happened to me as well. But with PyTorch version: 1.0.0

**Comment by suo** [comment]
This was fixed in https://github.com/pytorch/pytorch/pull/27566, which should go out as part of 1.4

**Comment by XinyingZheng** [comment]
> This was fixed in #27566, which should go out as part of 1.4

How to solve the problem, download Pytorch version 1.4？looking forward to your reply
